### PR TITLE
LPS-170572 | Add Autom. test ClayAlertStripe#CanDisplayCloseButton

### PR DESCRIPTION
**Pull request by deborasoliveira**
**ClayAlertStripe#CanDisplayCloseButton**
 **Test Plan**
- Given Clay Sample Portlet
- When the alert is a stripe variation
- Then the alert contains a close button

**JIRA Ticket**
https://issues.liferay.com/browse/LPS-170572

### CODE REVIEW

**Comment by deborasoliveira** [comment]
ci:test:frontend:nocompile

**Comment by deborasoliveira** [comment]
ci:test:sf

**Comment by deborasoliveira** [comment]
ci:test:frontend:nocompile

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/clay/ClayAlertStripe.testcase [hunk 1]
```diff
@@ -43,19 +43,20 @@ definition {
 		}
 	}
 
-    @description = "Verify stripe alert contains close button"
+	@description = "Verify stripe alert contains close button"
 	@priority = "5"
 	test CanDisplayAllAttributes {
```
Comment (john-co) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/clay/ClayAlertStripe.testcase]: typo: CanDisplayCloseButton

**Review by john-co — APPROVED** [review]
Looks good to me! :sunglasses:

Great job! Just one edit to the test name.
I'm also curious, can you share a screenshot of the stripe alerts on the clay sample portlet?

**Comment by deborasoliveira** [comment]
> Great job! Just one edit to the test name. I'm also curious, can you share a screenshot of the stripe alerts on the clay sample portlet?

screenshot-1671741050

**Comment by john-co** [comment]
ci:test:frontend

**Comment by john-co** [comment]
ci:help

**Comment by john-co** [comment]
ci:forward

**Pull request closed without merging**